spanner/google/cloud/spanner_v1/snapshot.py

```diff
@@ -99,10 +100,17 @@ def read(self, table, columns, keyset, index='', limit=0):
                       table's primary key
 
         :type limit: int
-        :param limit: (Optional) maximum number of rows to return
+        :param limit: (Optional) maximum number of rows to return.
+                      Incompatible with ``partition``.
+
+        :type partition: bytes
+        :param partition: (Optional) one of the partition tokens returned
+                          from :meth:`partition_read`.  Incompatible with
+                          ``limit``.
 
```

Comment:
Should ValueError be raised if both are specified?

Reply:
The server enforces that constraint.